fms/fms_io.F90

```diff
@@ -7895,11 +7896,12 @@ end subroutine get_mosaic_tile_file_ug
 
 
   !#############################################################################
-  subroutine get_mosaic_tile_grid(grid_file, mosaic_file, domain, tile_count)
+  subroutine get_mosaic_tile_grid(grid_file, mosaic_file, domain, tile_count, custom_path)
     character(len=*), intent(out)          :: grid_file
     character(len=*), intent(in)           :: mosaic_file
     type(domain2D),   intent(in)           :: domain
     integer,          intent(in), optional :: tile_count
+    character(len=*), intent(in), optional :: custom_path
```

Comment:
Please add a doxygen description of this new variable per the [style guide functions section](https://github.com/NOAA-GFDL/FMS/blob/main/CODE_STYLE.md#functions)

"Inline doxygen descriptions for all arguments, except the result variable."

Reply:
Will do. We'll have another PR coming up so will add it then.